src/vs/code/electron-sandbox/issue/issueReporterMain.ts

```diff
@@ -1071,9 +1066,10 @@ export class IssueReporter extends Disposable {
 	}
 
 	private updateProcessInfo(state: IssueReporterModelData) {
-		const target = document.querySelector('.block-process .block-info');
+		const target = document.querySelector('.block-process .block-info') as HTMLElement;
 		if (target) {
-			target.innerHTML = `<code>${state.processInfo}</code>`;
+			target.innerText = '';
+			target.appendChild(createHTMLElement('code', state.processInfo));
```

Comment:
The `innerText = ''; target.appendChild(...)` pattern might justify a new util-function, like `dom.ts#reset(target, newChild, ...moreChildren[])